src/navigation/Navigation.tsx

```diff
@@ -0,0 +1,47 @@
+//mport React, { useState } from 'react'
+import { AppBar, Toolbar, IconButton, Typography, Stack, Button, CssBaseline, useTheme, useMediaQuery } from '@mui/material';
+import CodeTwoToneIcon from '@mui/icons-material/CodeTwoTone';
+//import { ThemeProvider } from '@emotion/react';
+import { Link } from 'react-router-dom';
+import DrawerComponent from '../components/DrawerComponent';
+
+
+export default function Navigation() {
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down("md"));
```

Comment:
Tak się zastanawiałem jak ograłeś RWD, nawet nie wiedziałęm że material ma na to hooka :D nieźle